Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: help
+.PHONY: help frontend app-e2e
```

Comment:
Also I've just figure out why `frontend` acts like this - we have the dir with the same name.